src/current/v23.1/cdc-queries.md

```diff
@@ -100,7 +100,9 @@ You can **not** use the following functions with CDC queries:
 
 CDC queries allow you to customize your changefeed for particular scenarios. This section outlines several possible use cases for CDC queries.
 
-See [`CREATE CHANGEFEED`](create-changefeed.html) for examples on using the foundational syntax to create a changefeed. For information on sinks, see the [Changefeed Sinks](changefeed-sinks.html) page.
+CDC queries use `envelope=bare` message format by default. This omits the `after` key and stores any [metadata in the `crdb` field](#filter-columns). Depending on how you are filtering or adapting the message format with a CDC query and which sink you're emitting to, message format can vary from some of the example cases in this section.
```

Comment:
`omits the "after" key` could maybe be misread as saying that it omits the values that would be under the "after" key. I don't know a terse way of fixing that though, something more precise would be `puts the output of the SELECT clause at the top level of the message instead of under an "after" key`.

Reply:
I've adjusted this a little to hopefully get more clarity in this. Let me know if you think it needs some more adjustment.

Reply:
@HonoreDB Actually on this, I think I need to add more description... we only have the `crdb` field for CDC queries when there is additional information that cockroach is sending so like updated, resolved, otherwise the messages will actually be completely "bare" like 

```
{"city": "seattle", "type": "skateboard"}
{"city": "washington dc", "type": "bike"}
{"city": "washington dc", "type": "scooter"}
{"city": "washington dc", "type": "bike"}
{"city": "washington dc", "type": "skateboard"}
{"city": "washington dc", "type": "scooter"}
{"city": "washington dc", "type": "scooter"}
```

Reply:
I hadn't understood this, having just updated the example that contained the `updated` option... so will add to this now